website/docs/icons/usage-guidelines/partials/code/how-to-use.md

```diff
@@ -1,22 +1,21 @@
-There are multiple ways to use these icons in your codebase. The package can be installed as an [Ember addon](#ember-flight-icons) for the convenience of using a component with strong defaults, or it can be [consumed in React applications](#use-react) via direct import of the SVG file or as a standalone React/SVG icon component.
+## How to use icons
 
-## Use in Ember apps
+Helios icons can be used in many ways. The package can be installed as an [Ember addon](#ember-flight-icons) for the convenience of using a component with strong defaults. It can also be [consumed in React applications](#use-react) via direct import of the SVG file or as a standalone React/SVG icon component.
 
-### Installation
+### Using icons in Ember apps
+
+#### Installing the `ember-flight-icons` addon
 
-To install, run:
 ```bash
 yarn add @hashicorp/ember-flight-icons
 ```
 
 !!! Information
 
-**Note**
-
-Because this addon exposes a `data-test-icon` helper, it is suggested consumers install `ember-test-selectors`. [This Ember addon](https://github.com/simplabs/ember-test-selectors) strips out all `data-test-*` attributes for production builds.
+Because this addon exposes a `data-test-icon` helper, we recommend installing [`ember-test-selectors`](https://github.com/simplabs/ember-test-selectors). `ember-test-selectors` strips out all `data-test-*` attributes for production builds.
 !!!
 
-### Understanding the component
+#### Understanding the component
 
 The component comes with the following defaults:
```

Comment:
We _could_ use the `<Doc::ComponentApi>` component here. Might be good for consistency, but this is a simple way of capturing the properties

Reply:
I started to do that and realized that I didn't know all of the correct values to include so I left it alone. It would be a good future improvement, though. I'll add a Jira ticket and one of our engineers can pick it up post launch.